src/utils/display.tsx

```diff
@@ -10,14 +10,31 @@ export function displayTokenSymbolOrLink(token: TokenDetails): React.ReactNode |
   return displayName
 }
 
+/**
+ * computeMarketProp
+ * @description returns array of potentially accepted market names by:: SELLTOKEN <SEPARATOR> BUYTOKEN
+ * @param { sellToken, buyToken, acceptedSeparators: string[] }
+ */
 export function computeMarketProp({
   sellToken,
   buyToken,
+  acceptedSeparators = ['-', '/', ' '],
```

Comment:
I think, they usually don't use space as a separator. - and / are good, but the third I would say is empty string if that's easy, otherwise this is great already

![image](https://user-images.githubusercontent.com/2352112/85118474-ada21680-b220-11ea-824c-b80051b16f59.png)

Reply:
ok :+1: